packages/server/src/fhir/accesspolicy.ts

```diff
@@ -185,32 +187,20 @@ function addDefaultResourceTypes(resourcePolicies: AccessPolicyResource[]): void
 function applyProjectAdminAccessPolicy(
   project: Project,
   membership: ProjectMembership,
-  accessPolicy: AccessPolicy
-): AccessPolicy {
+  accessPolicy: PopulatedAccessPolicy
+): PopulatedAccessPolicy {
   if (project.superAdmin) {
-    // If the user is a super admin, then do not apply any additional access policy rules.
-    return accessPolicy;
-  }
-
-  if (accessPolicy) {
-    // If there is an existing access policy
-    // Remove any references to project admin resource types
+    for (const adminResourceType of projectAdminResourceTypes) {
+      if (!accessPolicy.resource.some((r) => r.resourceType === adminResourceType)) {
+        accessPolicy.resource.push({ resourceType: adminResourceType });
+      }
+    }
+  } else if (membership.admin) {
```

Comment:
Comment for myself to make sure we're preserving the current behavior.

```ts

export const projectAdminResourceTypes = [
  'PasswordChangeRequest',
  'UserSecurityRequest',
  'Project',
  'ProjectMembership',
  'User',
];
```

And we are adding each of those explicitly here 👍